js/CRM/Form/MembersOnlyEventTab.js

```diff
@@ -49,14 +72,22 @@ jQuery(document).ready(function(){
   }
 
   /**
-   * Shows/Hides the members-only events fields
+   * Shows/Hides the tab fields
    * based on 'Is members-only event ?' checkbox
    * value.
-   *
-   * @param isMembersOnlyEvent
    */
-  function toggleMembersOnlyEventFields(isMembersOnlyEvent) {
-    if (isMembersOnlyEvent){
+  function toggleTabFields() {
+    if (getIsMembersOnlyEventValue() || getIsGroupsOnlyEventValue()){
+      if (getIsMembersOnlyEventValue()) {
```

Comment:
I am not sure if it is a good idea to call getIsMembersOnlyEventValue() and getIsGroupsOnlyEventValue() twice? Should we assign them to local variable?   Something like? 
```


let isMembersOnlyEventValue = getIsMembersOnlyEventValue();
let IsGroupsOnlyEventValue() = getIsGroupsOnlyEventValue();
```